vignettes/summarizeOverlaps.Rmd

```diff
@@ -0,0 +1,328 @@
+---
+title: "Counting reads with `summarizeOverlaps`"
+author:
+- name: Valerie Obenchain
+- name: Rohit Satyam
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+clean: false
+date: "`r format(Sys.Date(), '%d %B, %Y')`"
+package: GenomicAlignments
+vignette: >
+  %\VignetteIndexEntry{Counting reads with summarizeOverlaps}
+  %\VignetteEncoding{UTF-8}  
+  %\VignetteDepends{GenomicAlignments, DESeq2, edgeR, rtracklayer}
+  %\VignetteKeywords{sequence, sequencing, alignments}
+  %\VignettePackage{GenomicAlignments}
+  %\VignetteAuthor{Valerie Obenchain}
+  %\VignetteEngine{knitr::rmarkdown}
+bibliography:
+- references.bib
+link-citations: TRUE
+reference-section-title: References
+
+output: 
+    BiocStyle::html_document:
+      toc_float: true
+      fig_caption: yes
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+```{r options, echo=FALSE, include=FALSE}
+knitr::opts_chunk$set(collapse = TRUE,eval = T)

```

Comment:
`eval`'s default is TRUE, so you don't have to set it. Also I advise against using `T` for `TRUE`.